<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_wresteling_participant</class>
 <widget class="QWidget" name="edit_wresteling_participant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>اعمال تغییرات</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="usename">
      <property name="placeholderText">
       <string>نام کاربری </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="pass">
      <property name="placeholderText">
       <string>رمز عبور</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="email">
      <property name="placeholderText">
       <string>@gmail.com</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="team_name">
      <property name="placeholderText">
       <string>نام تیم</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_8">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLineEdit" name="code_meli_2">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>کد ملی</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name_2">
      <property name="placeholderText">
       <string>نام ورزشکار</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
